From 8d9c2a6686b2998713122862e69f9093749c6e01 Mon Sep 17 00:00:00 2001
From: Yuval Mintz <Yuval.Mintz@cavium.com>
Date: Thu, 18 May 2017 15:13:36 +0300
Subject: qed: Don't log missing management statistics
References: bsc#1019695 FATE#321703 bsc#1019696 FATE#321702
Patch-mainline: Never, just a workaround upstream will contain different fix

Current code missess on filling statistics on behalf of management
for RDMA functions and logs that into system logs.
As Management firmware would request this periodically it creates
a continuous flood of prints.

Signed-off-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/qlogic/qed/qed_main.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/qlogic/qed/qed_main.c b/drivers/net/ethernet/qlogic/qed/qed_main.c
index 1e1d1db91e38..016c81c57817 100644
--- a/drivers/net/ethernet/qlogic/qed/qed_main.c
+++ b/drivers/net/ethernet/qlogic/qed/qed_main.c
@@ -1729,7 +1729,8 @@ void qed_get_protocol_stats(struct qed_dev *cdev,
 		qed_get_protocol_stats_iscsi(cdev, &stats->iscsi_stats);
 		break;
 	default:
-		DP_ERR(cdev, "Invalid protocol type = %d\n", type);
+		DP_VERBOSE(cdev, QED_MSG_SP,
+			   "Invalid protocol type = %d\n", type);
 		return;
 	}
 }
-- 
1.8.5.6

